README.md

```diff
@@ -1,7 +1,46 @@
 [![Build
 Status](https://travis-ci.org/final-ci/final-api.svg?branch=master)](https://travis-ci.org/final-ci/final-api)
 
-Final-API
-=========
+# final-api
 
-REST API for Final-CI.
+The API for final-CI. It is responsible for enqueing, monitoring and
+shutdown of tests.
+
+It features advanced query language to filter test results.
+
+It is connected to other supporting services, such as:
+* [node_starter](https://github.com/AVGTechnologies/node_starter)
+* [travis-test-results](https://github.com/final-ci/travis-test-results)
+* [test-aggregation](https://github.com/AVGTechnologies/test-aggregation)
+* [lab_manager](https://github.com/AVGTechnologies/lab_manager)
```

Comment:
If you mention lab manager i think you should mention travis-logs as well. But final-api is not directly connected to this two services.